src/components/filters/MeasureValueFilter/OperatorDropdownItem.tsx

```diff
@@ -48,6 +55,23 @@ export class OperatorDropdownItem extends React.PureComponent<IOperatorDropdownI
         onClick(operator);
         e.preventDefault();
     };
+
+    private renderBubble(message: string) {
+        return (
+            <div className="tooltip-bubble">
+                <BubbleHoverTrigger tagName={"div"} showDelay={400} hideDelay={200}>
+                    {this.renderTooltip()}
```

Comment:
There is no need to put it to separate function in my opinion.

Reply:
ok